src/sql/workbench/parts/query/browser/queryActions.ts

```diff
@@ -711,6 +724,11 @@ export class ListDatabasesActionItem implements IActionViewItem {
 	}
 
 	private getCurrentDatabaseName() {
```

Comment:
Could you update this to add the return type string | undefined while you're here?

Reply:
bump - it'd still be good to add the return type here

---
In reply to: [322376193](https://github.com/microsoft/azuredatastudio/pull/7084#discussion_r322376193) [](ancestors = 322376193)